persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/ClinicalAttributeMapper.java

```diff
@@ -18,4 +18,6 @@
     List<ClinicalAttributeCount> getClinicalAttributeCountsBySampleIds(List<String> studyIds, List<String> sampleIds);
 
     List<ClinicalAttributeCount> getClinicalAttributeCountsBySampleListId(String sampleListId);
+    
+    List<ClinicalAttribute> getClinicalAttributesByStudyIdsAndAttributeIds(List<String> studyIds, List<String> attributeIds, String projection);
```

Comment:
why need projection?

Reply:
Existing sql select statement needs `projection`. I'll have to update sql statement for getClinicalAttributesByStudyIdsAndAttributeIds to remove it

Reply:
I see. let's keep the way it is.

Reply:
Forgot to update. I remove that parameter